### ISSUE TITLE
Osoba odpowiedzialna w UE - jak skasować

### ISSUE THREAD

**Issue by allegrofan** [allegro/allegro-api#10562]
Cześć,
jak masowo skasować "Osoba odpowiedzialna w UE" we wszystkich ofertach skoro został podany Producent z siedzibą w UE?

**Comment by MaciejFrackowiak** [comment]
Nie zrobisz tego masowo korzystając z [PUT /sale/offer-modification-commands/{commandId}](https://developer.allegro.pl/documentation/#operation/modificationCommandUsingPUT). Wartość "null" możesz przekazać tylko pojedynczo działając w oparciu o [PATCH /sale/product-offers/{offerId}](https://developer.allegro.pl/documentation/#operation/editProductOffers).

**Comment by allegrofan** [comment]
> Nie zrobisz tego masowo korzystając z [PUT /sale/offer-modification-commands/{commandId}](https://developer.allegro.pl/documentation/#operation/modificationCommandUsingPUT). Wartość "null" możesz przekazać tylko pojedynczo działając w oparciu o [PATCH /sale/product-offers/{offerId}](https://developer.allegro.pl/documentation/#operation/editProductOffers).

w https://developer.allegro.pl/documentation/#operation/editProductOffers nie ma danych o nowym polu. Czy mogę przekazać tylko to jedno pole do zmiany w ofercie?

**Comment by MaciejFrackowiak** [comment]
Nie. Dla tablic takich jak m.in. productSet obowiązuje zasada "wszystko albo nic" - musisz przekazać pełną strukturę i jej zawartość dla productSet.
